plugins/@grouparoo/mailchimp/package.json

```diff
@@ -37,6 +37,7 @@
     "@types/node": "*",
     "actionhero": "25.0.8",
     "dotenv": "8.2.0",
+    "fs-extra": "9.1.0",
```

Comment:
Why was `fs-extra` added?  I don't see it in use in his PR.

Reply:
It's been used inside nockHelper. I saw the error when I ran the tests with newNock = true.